src/main/java/net/dv8tion/jda/api/entities/channel/concrete/PrivateChannel.java

```diff
@@ -26,6 +26,15 @@
 /**
  * Represents the connection used for direct messaging.
  *
+ * <p>This channel may communicate with different users in interactions triggered by user-installed apps:
+ * <ul>
+ *     <li>In bot DMs, this channel will send messages to the interaction caller.</li>
+ *     <li>In friend DMs, this channel will send messages to that friend,
```

Comment:
More accurately, the bot just cannot send messages in these channels, it can only reply to the interaction. I think we can describe it like that to make it easier to parse.